hotness/consumers.py

```diff
@@ -243,23 +252,68 @@ def _handle_anitya_update(self, upstream, package, msg):
                               "Skipping scratch build.")
                 return
 
-            self.log.info("Now with #%i, time to do koji stuff" % bz.bug_id)
+            cwd = os.getcwd()
+            result_rh = 0
+            rh_stuff = {}
             try:
-                # Kick off a scratch build..
-                task_id, patch_filename, description = self.buildsys.handle(
-                    package, upstream, version, bz)
-
-                # Map that koji task_id to the bz ticket we want to pursue.
-                self.triggered_task_ids[task_id] = bz
-
-                # Attach the patch to the ticket
-                self.bugzilla.attach_patch(patch_filename, description, bz)
-            except Exception as e:
-                heading = "Failed to kick off scratch build."
-                note = heading + "\n\n" + str(e)
-                self.log.warning(heading)
-                self.log.warning(traceback.format_exc())
-                self.bugzilla.follow_up(note, bz)
+                tmp = tempfile.mkdtemp(prefix='thn-rh', dir='/var/tmp')
+                result_rh, rh_stuff = self.buildsys.rebase_helper(package, upstream, tmp, bz)
+                if int(result_rh) == 0:
+                    self.log.info('Rebase package %s to %s was SUCCESSFULL' % (package, version))
+                    for number, reference in six.iteritems(rh_stuff['build_logs'],):
+                        note = 'Scratch build completed %s' % reference
```

Comment:
It looks like ``reference`` is a list, so that this ends up being printed in the ticket as ``['http://koji.fedoraproject.....']``.  Can you iterate over the list of references so that they are printed cleanly in the ticket?  Preferably each on a new line in the improbable case that there is more than one build log.